app/controllers/results_controller.rb

```diff
@@ -204,13 +206,13 @@ def show
         if assignment.assign_graders_to_criteria && current_user.ta?
           assigned_criteria = current_user.criterion_ta_associations
                                           .where(assessment_id: assignment.id)
-                                          .pluck(:criterion_type, :criterion_id)
-                                          .map { |t, id| "#{t}-#{id}" }
+                                          .pluck(:criterion_id)
+                                          .map(&:to_s)
```

Comment:
I think I changed this line in the merge from master, but we probably don't need the `to_s` now that we're working with ids. Try making `assigned_criteria` an array of ints, and modify how it's used below.